benchmarks/build.gradle

```diff
@@ -38,31 +39,52 @@ dependencies {
             libraries.mockito,
             libraries.hdrhistogram,
             libraries.netty_tcnative,
-            libraries.netty_epoll
+            libraries.netty_epoll,
+            libraries.math
 }
 
 configureProtoCompilation()
 
+def vmArgs = [
+        "-server",
```

Comment:
Yeah, inclined to agree if only because we don't want our benchmarks to appear too exotic